docs/how-to-guides/gitops-using-azure-arc-onboarding-component.md

```diff
@@ -0,0 +1,238 @@
+weight: 10
+---
+
+## Introduction
+
+Azure Arc offers simplified management, faster app development, and consistent Azure services.
+
+With Azure Arc, you can:
+
+- Centrally manage a wide range of resources including
+[Windows](https://azure.microsoft.com/en-in/campaigns/windows-server/) and
+[Linux](https://azure.microsoft.com/en-in/overview/linux-on-azure/) servers, SQL server,
+[Kubernetes](https://azure.microsoft.com/en-in/services/kubernetes-service/) clusters and [Azure
+services](https://azure.microsoft.com/en-in/services/azure-arc/hybrid-data-services/).
+
+- Establish central visibility in the [Azure portal](https://azure.microsoft.com/en-in/features/azure-portal/)
+and enable multi-environment search with Azure Resource Graph.
+
+- Meet [governance](https://azure.microsoft.com/en-in/solutions/governance/) and compliance standards for
+apps, infrastructure and data with [Azure Policy](https://azure.microsoft.com/en-in/services/azure-policy/).
+
+- Delegate access and manage security policies for resources using role-based access control (RBAC) and [Azure
+Lighthouse](https://azure.microsoft.com/en-in/services/azure-lighthouse/).
+
+- Organise and inventory assets through a variety of Azure scopes, such as management groups, subscriptions,
+resource groups and tags.
+
+This guide provides the steps for onboarding a Lokomotive cluster on Azure Arc.
+
+## Learning objectives
+
+At the end of this tutorial you will have onboarded a Lokomotive cluster on Azure Arc, create an Azure Arc
+Kubernetes configuration and deploy configuration of a sample Nginx Deployment using GitOps.
+
+## Prerequisites
+
+* A Lokomotive cluster.
+
+* Azure command-line interface `az` installed on the local machine.
+
+* `jq` installed on the system.
+
+* Register providers for Azure Arc enabled Kubernetes:
+   ```bash
+   az provider register --namespace Microsoft.Kubernetes
+   az provider register --namespace Microsoft.KubernetesConfiguration
+   az provider register --namespace Microsoft.ExtendedLocation
+   ```
+
+   Monitor the registration process. Registration may take up to 10 minutes.
+
+   ```bash
+   az provider show -n Microsoft.Kubernetes -o table
+   az provider show -n Microsoft.KubernetesConfiguration -o table
+   az provider show -n Microsoft.ExtendedLocation -o table
+   ```
+
+   **NOTE**: This registration is needed only once per tenant.
+
+* Install `k8s-configuration` extension for Azure cli.
+
+  ```bash
+  az extension add --name k8s-configuration
+  ```
+
+* Create a resource group:
+
+   ```bash
+   RG_NAME="AzureArcTest"
+   az group create --name "${RG_NAME}" --location EastUS --output table
+   ```
+
+* Create a service principal, its credentials and assign roles to the service principal.
+
+  ```bash
+  # Create a ServicePrincipal and its credentials.
+  SP_NAME=azure-arc-onboarding-service-principal
+  az ad sp create-for-rbac -n "${SP_NAME}" --skip-assignment -o jsonc > /tmp/sp-cred.json
```

Comment:
Maybe use `mktemp` to generate a random file?

Reply:
I think its not needed